app/models/rule.rb

```diff
@@ -3,8 +3,8 @@ class Rule < ApplicationRecord
   has_one :account, through: :account_user
 
   validates :spending_limit, numericality: {
-                                             # less_than_or_equal_to: account.balance,
-                                             greater_than_or_equal_to: 0
-                                           },
+    # less_than_or_equal_to: account.balance,
+    greater_than_or_equal_to: 0
+  },
                              allow_nil: true
```

Comment:
@swindler I am not responsive for rules, only fixed syntax. Let me avoid code changes in it.